lg69t/README.md

```diff
@@ -16,21 +16,22 @@ python3 -m pip install -r requirements.txt
 
 And download the latest firmware package from [Point One's Developer Portal](https://pointonenav.com/docs/).
 
-
 ## Application and GNSS
 
-To update the Application and GNSS firmware:
+To update the Application and GNSS firmware from previous Point One Firmware, you must be use the Standard COM Port (UART1, typically /dev/ttyUSB1 in Linux for P1SDK):
 
 ```
-python3 firmware_tool.py --port=/dev/ttyUSB0 --app=/path/to/quectel-lg69t-am-0.XX.0_upg.bin
-python3 firmware_tool.py --port=/dev/ttyUSB0 --gnss=/path/to/lg69t_teseo_A.B.CC.D_sta.bin
+python3 firmware_tool.py --port=/dev/ttyUSB1 --app=/path/to/quectel-lg69t-am-0.XX.0_upg.bin
+python3 firmware_tool.py --port=/dev/ttyUSB1 --gnss=/path/to/lg69t_teseo_A.B.CC.D_sta.bin
 ```
 
-Replace `/dev/ttyUSB0` with the serial port connected to the device (use the appropriate COM port number in Windows, e.g., `COM1`).
+Replace `/dev/ttyUS1` with the serial port connected to the device (use the appropriate COM port number in Windows, e.g., `Standard COM Port(COM2)`).
+
+Replace `/path/to/quectel-lg69t-am-0.XX.0_upg.bin` and `/path/to/lg69t_teseo_A.B.CC.D_sta.bin` with the path to the application image and GNSS image files respectively. The device must be rebooted after each command is executed successfully.
 
-Replace `/path/to/quectel-lg69t-am-0.XX.0_upg.bin` and `/path/to/lg69t_teseo_A.B.CC.D_sta.bin` with the path to the application image and GNSS image files respectively.
+## Considerations
 
-If the board is not currently running valid application code, `upgrade_test.py` should be run with the `--manual-reboot` flag. In this case, immediately after running the script the board should be manually power cycled.
+If the device is not running legacy Point One application code, the `firmware_tool.py` should be run with the `--manual-reboot` flag. In this case, manually power cycle the board after running the script.
```

Comment:
This doesn't seem right to me. "not running legacy code ... should use --manual-reboot" - meaning if it is running _current_ code, you _must_ use that flag, but if it is running _old_ code you should _not_ use that flag? I think that's backwards/not corret?

`--manual-reboot` tells the tool "when the firmware tool tries to reboot the device, the device won't respond to the request." That should only happen if the device is running very _old_ firmware, or if the firmware is not working at all currently.

Reply:
I think the statement reads as: If not using legacy P1 code (i.e. bootloader just flashed, legacy China firmware, etc.), then you must use the flag.

Reply:
Ok, that's really confusing I think. When I think "P1 code" I think the software version like AM 0.15.0 or something. The bootloader is really a totally separate thing, and most users won't know anything about which bootloader version they have. The bootloader does not currently have a way to query its version. Plus, the new bootloader that does not require the manual reboot has not been released yet.

Reply:
You are thinking about it correct, P1 code is AM, AP, or AH firmware. this is saying if none of those are loaded, then you need to use the flag.

Agreed it's niche, but for customers like Pronto and Amazon, they may need to know if they are need to flash a brand new board.

Reply:
Ohhh ok. I thought you were talking about "if the device is running an old bootloader". I think the word "legacy" is maybe what's confusing here. Saying "not running legacy code" implies that it is running new code. We should say "if this is a brand new device that has not been pre-loaded with Point One firmware..."

Reply:
Honestly I don't think you need the "why" here. Just be like "if the reset command is not working, you can use the manual reset mode"

Reply:
Ok, updating shortly.